lib/advisory.js

```diff
@@ -242,7 +242,7 @@ class Advisory {
     // check the dependency of this version on the vulnerable dep
     // if we got a version that's not in the packument, fall back on
     // the spec provided, if possible.
-    const mani = this[_packument].versions[version] || {
+    const mani = this[_packument].versions && this[_packument].versions[version] ? this[_packument].versions[version] : {
```

Comment:
hi @yadirhb, thanks a lot for contributing to this issue, if possible could you try to fix this linting issue so git flow will pass?

Reply:
It just needs line-breaks:

```
const mani = this[_packument].versions && this[_packument].versions[version] ? 
    this[_packument].versions[version] : 
    {
```

I submitted [this PR](https://github.com/yadirhb/metavuln-calculator/pull/1) a few days ago to address that. Should I setup a new one here to get this moving?

Reply:
https://github.com/yadirhb/metavuln-calculator/pull/1/files#r1147585254

Reply:
tbh I'm not sure if original author is available to accept your changes or if opening a new PR would help :/